src/Test/Perf/ConsumptionParser/Program.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Roslyn.Test.Performance.Utilities.ConsumptionParser
+{
+    class Program
+    {
+        public static void Main()
+        {
+            const string prefix = "PerfResults-";
+            const string share = @"\\mlangfs1\public\basoundr\PerfTraces\";
+
+            var runs = new List<Tuple<string, ConsumptionParse, RunInfo>>();
+
+            // Collect run information from the share
+            foreach (var directory in Directory.GetDirectories(share))
+            {
+                var lastPath = directory.Substring(share.Length);
+                if (!lastPath.StartsWith(prefix))
+                {
+                    continue;
+                }
+
+                var date = lastPath.Substring(prefix.Length);
+                var consumptionXml = Path.Combine(directory, "ConsumptionTempResults.xml");
+                var resultJson = Directory.EnumerateFiles(directory, "Roslyn*.json").First();
```

Comment:
Which one are we picking here?  If we only expect 1, I'd suggest ```Single```.  If there may be more than one, it would be good to at least add a comment about why it's ok to just pick First.